File: CPP/8 Kyu/remove_exclamation_marks.cpp
#include <string>
using namespace std;
string removeExclamationMarks(string s){
  string copy = ""; 
  int tam = s.length();
  for(int i = 0; i<tam; i++){
    if(s[i] != '!'){
        copy+=s[i];
    }
  }
  
  return copy;
}

File: Rust/8 Kyu/remove_exclamation_marks.rs
fn remove_exclamation_marks(input: &str) -> String {
    let mut copy = String::from("");
    
    for x in input.chars(){
        if x != '!'{
            copy  = copy.to_owned() + &String::from(x);
        }  
        
    }
        
    copy    
}